Library/Formula/qt5.rb

```diff
@@ -72,6 +74,13 @@ class Qt5 < Formula
   depends_on OracleHomeVarRequirement if build.with? "oci"
 
   def install
+    # Save users some frustration by telling them that `--cc=gcc-5` or similar
+    # won't work. It won't be picked up by the Qt build system, there is no easy
+    # way to select an arbitrary compiler, and it's not supported anyway.
+    if ENV.compiler != :clang
```

Comment:
That's OK, don't worry. I think probably `fails_with :gcc => "5"` with a decent enough message will be enough for now.

> I came to this realization and believe it to be true after looking at Qt's mkspecs and after some personal experiments. Notice how ??.make.cc doesn't exist or is very small in a Qt build as it doesn't use the compiler wrappers provided by superenv.

When you get bored making Qt respect superenv seems like a good wee project.